src/botPage/view/style.js

```diff
@@ -73,9 +73,9 @@ export const bottomWarning = {
 
 export const bottomWarningLink = { textDecoration: 'underline' };
 
-export const green = { color: 'green' };
+export const green = { color: 'green', float: 'left' };
```

Comment:
By adding this `float` the `totalWins` and `totalLosses` cells in the account overview at the bottom of the table are incorrectly aligned.

Reply:
Okay, I'll make separate style variables for each